### ISSUE TITLE
LARMOR: homing on sample changer axis

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#4713]
As a scientist on LARMOR I would like the homing on my sample changer translation axis to be sorted out.

Apparently "it has never worked" but Rob wants it working today.

@aaron-long and @davidkeymer are going down to LARMOR to look at this (thank you)

**Comment by aaron-long** [comment]
David and I went down to LARMOR to assist. We were also joined by Ian.

R had noted that they had tried to set motor controller up using their known historic values, but the axis was moving in the wrong direction and would hit the axis limits when attempting to drive to known motion set points. We homed the axis and tested it's limits, finally defining a new off-set that was significantly different than the one that had been set for the axis.

I double checked the deployment pv motor record values for the axis (MTR0703) and the historic value was -399.7 (rather than the ~-1136 set).

We updated the settings and tested that the motion set points would drive to their desired positions and limits.
